DSLink.Tests/NodeTests.cs

```diff
@@ -0,0 +1,239 @@
+﻿using DSLink.Connection;
+using DSLink.Container;
+using DSLink.Nodes;
+using DSLink.Respond;
+using Moq;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace DSLink.Tests
+{
+    [TestFixture]
+    public class NodeTests
+    {
+        private Mock<AbstractContainer> _mockContainer;
+        private Mock<Connector> _mockConnector;
+        private Mock<Responder> _mockResponder;
+        private Mock<SubscriptionManager> _mockSubManager;
+        private Node _superRootNode;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockContainer = new Mock<AbstractContainer>();
```

Comment:
It's the interface for DSLinkContainer, but I think it might be a good idea to just drop it, since it really isn't useful for testing because we have mocking(which I wasn't familiar with when I originally thought about the design).